src/cli/cli.rs

```diff
@@ -416,6 +417,24 @@ impl CliConfig {
     }
 }
 
+/// Type of the "format" argument.
+// It is an enum that maps 1:1 with the formats defined in events library.
+#[derive(Debug, Default, Clone, Copy, Eq, PartialEq, ValueEnum)]
+pub(crate) enum CliDisplayFormat {
+    SingleLine,
+    #[default]
+    MultiLine,
+}
+
+impl CliDisplayFormat {
+    pub(crate) fn format(&self) -> DisplayFormat {
```

Comment:
Would be better to implement `Into` here. Would also make the `format.format()` calls a bit nicer.

Reply:
Agree. Way better